src/components/screens/FormScreen.tsx

```diff
@@ -111,10 +111,8 @@ function FormScreen() {
 
 export default FormScreen;
 
-function randomizer(
-  formMethods: UseFormReturn<PaniniFormData, any, undefined>
-) {
-  let randomForm: PaniniFormData = {
+function randomForm() {
+  const randomForm: PaniniFormData = {
```

Comment:
I'd like to point out in here, that PaniniFormData is an interface and not a Class with a Constructor, therefore I cannot create a new instance of it using "new" keyword. If you have any better way to instantiate this, please tell me because I feel like I was forced to do it that way.

Reply:
Hmm, Im not sure I understand. Why would you initialize new instance of something? It's good that it's an interface ;D